src/inverseRobot.js

```diff
@@ -24,7 +24,17 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobots = {};
+
+  for (const [key, value] of Object.entries(robot)) {
+    if (value in inversedRobots) {
```

Comment:
you can use hasOwnProperty

Reply:
Agree. Use `hasOwnProperty` instead `in`

Reply:
Thank you, resolve)